3135/3/views/restaurant_in_detail.erb

```diff
@@ -0,0 +1,43 @@
+<div>
+    <hr>
+    <h4><%= @restaurant.name %></h4>
+    <p><%= @restaurant.location %></p>
+    <p><%= @restaurant.description %></p>
+    <h4>Average rating: <%= @average_rating.to_s %></h4>
```

Comment:
Expression tags converts to a string before rendering, noneed to use to_s method